sys/console/full/src/console.c

```diff
@@ -116,7 +116,11 @@ console_read(char *str, int cnt, int *newline)
     }
     cmd = ev->ev_arg;
     len = strlen(cmd->line);
-    strncpy(str, cmd->line, len+1);
+    if (cnt < len) {
+        len = cnt;
```

Comment:
That would prevent the unintended buffer overrun, but then, if we are being pedants, the docs require a change as cnt is no longer the number of chars *coppied* when `cnt - 1 < len`. If we just change the word *coppied* to *written* in the docs that might work? Thanks!